demos/demo_simulate_solo12.py

```diff
@@ -17,11 +17,17 @@
 
 if __name__ == "__main__":
 
-    # Create a Pybullet simulation environment
+# ! Create a Pybullet simulation environment before any robots !
     env = BulletEnvWithGround()
 
-    # Create a robot instance. This initializes the simulator as well.
-    robot = env.add_robot(Solo12Robot)
+    # Create a robot instance. This adds the robot to the simulator as well.
+    robot = Solo12Robot(useFixedBase=False)
```

Comment:
The `useFixedBase=False` is the default, no? So maybe we can just remove it to make the constructor shorter.